cpp-ch/local-engine/Parser/aggregate_function_parser/CommonAggregateFunctionParser.cpp

```diff
@@ -25,17 +25,17 @@ REGISTER_COMMON_AGGREGATE_FUNCTION_PARSER(Sum, sum, sum)
 REGISTER_COMMON_AGGREGATE_FUNCTION_PARSER(Avg, avg, avg)
 REGISTER_COMMON_AGGREGATE_FUNCTION_PARSER(Min, min, min)
 REGISTER_COMMON_AGGREGATE_FUNCTION_PARSER(Max, max, max)
-REGISTER_COMMON_AGGREGATE_FUNCTION_PARSER(StdDev, stddev, stddev_samp)
```

Comment:
How much performance degradation if replace them all to stable implementations?  

According to https://clickhouse.com/docs/en/sql-reference/aggregate-functions/reference/stddevsamp: use the stddevSampStable function. It works slower but provides a lower computational error.